src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import duke.command.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructs a Duke chatbot. The TaskList will be loaded from
+     * the Storage at the specified file path during this stage.
+     *
+     * @param filePath the specified file path parameter.
+     */
+    public Duke(String filePath) {
+        this.ui = new Ui();
```

Comment:
Might want to remove `this` if it is clear which variable is being referred to.

Reply:
Thanks for the suggestion! Resolved in dfffdfb5c287aecd0c382ca6f3eeb96ad1a39af6.